lineage/states/stateCommon.py

```diff
@@ -75,6 +84,42 @@ def gamma_estimator(gamma_obs, time_cen, gammas, constant_shape, x0):
 
     return xOut
 
+def gamma_estimator_atonce(gamma_obs, time_cen, gamas):
```

Comment:
You know what the inputs and outputs to this should look like. This should be written with a separate branch and test.

Reply:
Moved to another branch.